changelog/17.0/17.0.0/summary.md

```diff
@@ -161,6 +161,12 @@ This feature enables the following flow:
     ```
  2. Restore that backup with a mere `Restore` command, without having to specify `--external-decompressor`.
 
+#### <a id="throttler-config-via-topo" /> vttablet --throttler-config-via-topo
```

Comment:
This also needs to be added to the ToC at the top.

Reply:
done